src/sys/mod.rs

```diff
@@ -54,6 +54,7 @@ cfg_os_poll! {
 #[cfg(unix)]
 cfg_os_poll! {
     mod unix;
+    #[warn(unused_imports)]
```

Comment:
This doesn't make sense. Did you mean to disable the lint? That would be the following, but really we shouldn't need it at all.

```suggestion
    #[allow(unused_imports)]
```

Reply:
Yes, `#[allow(unused_imports)]` does make sense. Fixed.

Reply:
@Thomasdezeeuw can you please merge it now?